test/k4FWCoreTest/options/CheckOutputFiles.py

```diff
@@ -58,6 +58,17 @@ def check_events(filename, number):
         raise RuntimeError("Number of events does not match expected number")
 
 
+def check_metadata(filename, keys, values):
+    print(f'Checking file "{filename}" for metadata')
+    podio_reader = podio.root_io.Reader(filename)
+    metadata = podio_reader.get("metadata")[0]
+    for key, value in zip(keys, values):
```

Comment:
```suggestion
    for key, value in expected_metadata.items():
```
Especially since the changes to this function are minimal.